modules/debts_api/lib/debts_api/v0/fsr_form_transform/enhanced_expense_calculator.rb

```diff
@@ -94,11 +98,14 @@ def default_expenses
         end
 
         def get_rent_mortgage_expenses
-          safe_number(@old_rent_mortgage_attr)
-          rent_or_mortgage_expenses = @expense_records.filter do |record|
-            [RENT, MORTGAGE_PAYMENT].include?(record['name'])
+          if @expense_page_active
```

Comment:
in early return might be nice here too like this:
`return safe_number(@new_rent_mortgage_attr) if @expense_page_active`